examples/carousel/js/carousel-tablist.js

```diff
@@ -30,21 +30,20 @@ var CarouselTablist = function (node, options) {
   this.tabpanelNodes = [];
 
   this.liveRegionNode = node.querySelector('.carousel-items');
-  this.pauseButtonNode = null;
+  this.pausePlayButtonNode =  document.querySelector('.carousel-tablist .controls button.rotation');
 
   this.playLabel = 'Start automatic slide show';
   this.pauseLabel = 'Stop automatic slide show';
 
   /* State properties */
-  this.forcePlay = false; // set once the user activates the play/pause button
-  this.playState = !options.paused; // state of the play/pause button
-  this.rotate = !options.paused; // state of rotation
+  this.hasUserActivatedPlay = false; // set when the user activates the play/pause button
+  this.isAutoRotationDisabled = options.norotate // This property for disabling auto rotation
+  this.isPlayingEnabled = !options.paused; // This property is also set in updatePlaying method
```

Comment:
Wow thanks so much for following up with this PR to match the variable names to https://github.com/w3c/aria-practices/pull/1387!